docs/data/material/getting-started/templates/checkout/ToggleColorMode.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { PaletteMode } from '@mui/material';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+
+import WbSunnyRoundedIcon from '@mui/icons-material/WbSunnyRounded';
+import ModeNightRoundedIcon from '@mui/icons-material/ModeNightRounded';
+
+interface ToggleColorModeProps {
+  mode: PaletteMode;
+  toggleColorMode: () => void;
+}
+
+function ToggleColorMode({ mode, toggleColorMode }: ToggleColorModeProps) {
+  return (
+    <Box sx={{ maxWidth: '32px' }}>
+      <Button
+        variant="text"
+        onClick={toggleColorMode}
+        size="small"
+        aria-label="button to toggle theme"
+        sx={{ minWidth: 32, height: 32, p: 1 }}
+      >
+        {mode === 'dark' ? (
+          <WbSunnyRoundedIcon fontSize="small" />
+        ) : (
+          <ModeNightRoundedIcon fontSize="small" />
+        )}
+      </Button>
+    </Box>
```

Comment:
```suggestion
    <IconButton
      onClick={toggleColorMode}
      color="primary"
      aria-label="Theme toggle button"
    >
      {mode === 'dark' ? (
        <WbSunnyRoundedIcon fontSize="small" />
      ) : (
        <ModeNightRoundedIcon fontSize="small" />
      )}
    </IconButton>
```

We can use an Icon Button here instead — this will save us from forcing the button to be rounded with fixed height and width, and remove the extra unnecessary Box.